rskj-core/src/main/java/co/rsk/peg/Federation.java

```diff
@@ -76,7 +74,7 @@ public List<BtcECKey> getBtcPublicKeys() {
     }
 
     public int getNumberOfSignaturesRequired() {
-        return members.size() / 2 + 1;
+        return getMembers().size() / 2 +1 ;
```

Comment:
Did you choose to call getMembers instead of members for any particular reason? Just asking because I see a mix of both on this class so we should probably settle for one option

Reply:
nope, my bad. thanks!